pkg/controller/controllercmd/cmd.go

```diff
@@ -39,23 +39,42 @@ type ControllerCommandConfig struct {
 
 	basicFlags *ControllerFlags
 
+	controllerBuilderModifier ControllerBuilderModifier
+
 	// DisableServing disables serving metrics, debug and health checks and so on.
 	DisableServing bool
 
 	// DisableLeaderElection allows leader election to be suspended
 	DisableLeaderElection bool
 }
 
+// ControllerBuilderOptions contain data needed for ControllerBuilder modification
+type ControllerBuilderOptions struct {
```

Comment:
This seems a bit excessive, I'd imagine tweaking only the bits you require see https://github.com/openshift/cluster-policy-controller/commit/a94cb90b7397b26d1fc72c2ffaa29d27d41a541e#r53164748 where I'm pointing only the necessity to translate from one type of config to another, wdyt?